### PR TITLE
Update server versions on frontpage

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,7 +232,7 @@ <h2>Nextcloud 28<a class="headerlink" href="#nextcloud-latest" title="Permalink
 							</div>
 
 							<div class="section" id="nextcloud-stable">
-								<h2>Nextcloud 27<a class="headerlink" href="#nextcloud-stable" title="Permalink to this headline">¶</a></h2>
+								<h2>Nextcloud 28<a class="headerlink" href="#nextcloud-stable" title="Permalink to this headline">¶</a></h2>
```
Comment (blizzz) [inline index.html:235]: after release?

Comment (nickvergessen) [reply index.html:235]: well after branch off I guess we would like to make clear that "latest" is not "28" anymore.

**Comment by blizzz** [comment]
Please don't forget to set the milestone when opening a PR :smiley:

**Review by blizzz — APPROVED** [review]

**Comment by tflidd** [comment]
@blizzz Why did you guys merge this already, NC 28 was not released yet (https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and this creates confusion https://help.nextcloud.com/t/where-is-nc28-update/176198/3 ?

**Comment by nickvergessen** [comment]
Because we don't like to touch things multiple times. As soon as we branch off stableX from the main development branch we need to build a different documentation version.

That being said iirc even after Tuesday there could be such questions as we have chunked rollouts and not everyone will receive the 28 release via the updater on Tuesday but only roughly 30%

We do it like this since ~20 versions and now one person for confused. Sounds good to me

**Pull request merged**